Automattic/wp-calypso
# Remove the Free Plans Domain Upsell

#### Changes proposed in this Pull Request

Remove the Domain Upsell from the launch-site flow

The upsell moved some important stats in the right direction but a subsequent analysis revealed that it moved other stats in the opposite direction so the overall impact is questionable.

Check the latest comments on pcvIcj-2A-p2 for details on the rationale behind this change.

This is the step being removed:

<img width="1146" alt="Screenshot 2021-02-18 at 16 44 44" src="https://user-images.githubusercontent.com/82778/108373745-e4de2280-7208-11eb-8527-7ca65615aad1.png">

#### Testing instructions

1. Create a new user and site with the /start/user flow. It should work
2. From /home click the Launch button
3. Select a free .wordpress.com domain and a free plan
4. The site should be launched, no step 4 offering a domain without a plan should be visible

See the original #47963. This is not a revert because there were other changes between the free experiment and now.
